### DISCUSSION TITLE
Invalid <Link> with <a> child error when fetching Wordpress posts via GraphQL in Next.js

### DISCUSSION THREAD

**Discussion by stevenoism** [vercel/next.js#45031] [Help]
0

I am trying to fetch Wordpress posts via graphql in my Next.js application, using the git repository "https://github.com/vercel/next.js/tree/canary/examples/cms-wordpress".

I am getting the following error: "Error: Invalid with child. Please remove or use .

![image](https://user-images.githubusercontent.com/123053842/213367682-d6c55904-54e4-4774-9115-1d8aed9bf4e8.png)

Learn more: https://nextjs.org/docs/messages/invalid-new-link-with-extra-anchor This error happened while generating the page. Any console logs will be displayed in the terminal window."

I don't know what is causing this error or how to fix it. Could someone please help me understand what is causing this issue and how to resolve it?

**Comment by icyJoseph** [comment]
Hi,

This is because in Next 13, a new link behaviour was introduced, and not all examples are fixed just yet.

A quick fix, create a file `next.config.js` at the root of your project and put this in

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  swcMinify: true,
  experimental: {
    newNextLinkBehavior: false,
  },
};

module.exports = nextConfig;
```

This will opt-out from the new link behaviour.

The proper fix is to find every:

```jsx
<Link href="/">
   <a>text</a>
</Link>
```

And change it to:

```jsx
<Link href="/">
    text
</Link>
```

## Edit

That example has been updated... So maybe the error is within a library or your own code changes?
